client/cli/src/params/import_params.rs

```diff
@@ -186,3 +154,23 @@ pub struct ExecutionStrategiesParams {
 	)]
 	pub execution: Option<ExecutionStrategy>,
 }
+
+impl ExecutionStrategiesParams {
+	/// Check if one of the parameters is still passed and print a warning if so.
+	fn check_usage_and_print_deprecation_warning(&self) {
+		for (param, name) in [
+			(&self.execution_syncing, "execution-syncing"),
+			(&self.execution_import_block, "execution-import-block"),
+			(&self.execution_block_construction, "execution-block-construction"),
+			(&self.execution_offchain_worker, "execution-offchain-worker"),
+			(&self.execution_other, "execution-other"),
+			(&self.execution, "execution"),
+		] {
+			if param.is_some() {
+				eprintln!(
+					"CLI parameter `--{name}` is deprecated and will be removed in the future!"
```

Comment:
```suggestion
					"CLI parameter `--{name}` has no effect anymore and will be removed in the future!"
```

Something like this would be better I think. (`deprecated` might suggest that it still does something; if we say `no effect` or something along these lines then it should be obvious that it doesn't do anything anymore)

Also, perhaps a `log::warn!` would be better instead of `eprintln!`? (Since warnings should be always enabled by default anyway.)